tests/gateway/test_gateway_app.py

```diff
@@ -59,6 +59,16 @@ def test_health(client: TestClient):
     assert response.json() == {"status": "OK"}
 
 
+def test_favicon(client: TestClient):
+    response = client.get("/favicon.ico")
+    assert response.status_code == 200
```

Comment:
We might move the favicon to a different folder in the future. This test tells us if we do that.